promscale/installation/helm.md

```diff
@@ -66,19 +69,16 @@ can provide the database URI, or specify connection parameters.
     ```bash
     helm repo update
     ```
-1.  Install the Promscale Helm chart:
-    ```bash
-    helm install --name my-release .
-    ```
-1.  Open the `values.yaml` configuration file, and locate the `connection`
+1.  Open the `values.yaml` configuration file, download the tobs
+    `values.yaml` from [here][tobs-values-yaml] and locate the `connection`
     section. Add or edit this section with your TimescaleDB connection details:
     <terminal>
 
     <tab label='Database URI'>
 
     ```yaml
     connection:
-      uri: <TIMESCALE_DB_URI>
+      uri: postgres://<username>:<password>@<host>:<port>/<dbname>?sslmode=require
     ```
 
     </tab>
```

Comment:
In the connection settings below, please replace `db.timescale.svc.cluster.local` with `timescaledb.default.svc.cluster.local`

Reply:
@VineethReddy02 @Loquacity this comment was not yet addressed.